### DISCUSSION TITLE
Use different colors when plotting the Ichimoku Cloud?

### DISCUSSION THREAD

**Discussion by Borromeow** [highfestiva/finplot#238] [Q&A]
I am having trouble with the filling colors for the Ichimoku cloud. I would like to fill the cloud with green when the leading Span A is over the leading span B, and red when the leading span A is under the leading span B.

I've seen that with matplotlib is possible to do this with the creation of two separate strings of code, using inside the function fill_between (same as in finplot) a 'when'. 

I've tried to replicate it in finplot, but when I run the program I get a TypeError saying that "fill_between() got an unexpected keyword argument 'where' "

Am I missing something?

btw this library rocks!!

**Comment by highfestiva** [comment]
Yes, there is no such thing as "where" in finplot's fill between. You unfortunately have to make two separate plots: one for the positive and one for the negative.
